.github/workflows/scheduled-run.yml

```diff
@@ -0,0 +1,16 @@
+name: scheduled-run
+
+on:
+  schedule:
+    - cron:  '*/15 * * * *'
+
+jobs:
+  scheduled-run:
+    runs-on: ubuntu-latest
+    container: quay.io/enarx/fedora
```

Comment:
We probably don't want to use our container for this. It is rather large.

Reply:
I suppose the only things we really need are Python and PyGithub. Makes sense to have a container with only those things included.

Reply:
If `ubuntu-latest` has `PyGithub`, we probably don't need a container at all.

Reply:
Very good point. At worst, we'd probably have to `pip install` it, which would still take less time than spinning up a container would.